app/controllers/Application.java

```diff
@@ -1816,6 +1816,39 @@ public static Result getTuningParameter(String jobId) {
     }
   }
 
+  public static Result getSparkJobBestParamter(String jobDefId) {
```

Comment:
Change function name to getJobBestParameter

Reply:
Done.